### PR TITLE
tweak docs style

### PR DESCRIPTION
**Pull request by wozeparrot**
I do like this more, not sure about you @chenyuxyz

### CODE REVIEW

**Comment by chenyuxyz** [comment]
sure was testing something similar

btw do you know how to change favicon color based on dark / light mode? white favicon is bad with light mode broswer

File: mkdocs.yml [hunk 1]
```diff
@@ -3,17 +3,19 @@ site_name: tinygrad docs
...
+  - index.md
+  - Quickstart: quickstart.md
+  - Showcase: showcase.md
+  - MNIST Tutorial: mnist.md
+  - API Reference:
+    - Tensor: tensor.md
+    - Function: function.md
+    - dtypes: dtypes.md
+    - nn (Neural Networks): nn.md
+  - Notes:
```
Comment (chenyuxyz) [inline mkdocs.yml:16]: Notes is meh... env var should be in API and developer can have its own section?

Comment (wozeparrot) [reply mkdocs.yml:16]: yea i didn't really know where to put these two

**Comment by wozeparrot** [comment]
don't think its possible to change favicon based on light or dark mode, also the browser tab bar color isn't always determined by the site theme color or even the system color preference.

**Comment by wozeparrot** [comment]
oh it is possible if we use a svg favicon

**Pull request merged**